comment-snyk-report/src/types.ts

```diff
@@ -0,0 +1,63 @@
+// SnykTestReportVulnerability and SnykTestReport are the partial types, for the full types see the Snyk documentation
```

Comment:
Let's add a link to which part of the docs, if you have a particular page in mind.